apps/climatemappedafrica/src/components/HURUmap/Panel/DesktopPanel/DesktopPanel.js

```diff
@@ -41,29 +47,24 @@ function DesktopPanel({ sx, ...props }) {
             overscrollBehaviorBlock: "none",
           },
         }}
+        variant="persistent"
         PaperProps={{
           elevation: 0,
           square: true,
-          sx: ({ typography }) => ({
+          sx: () => ({
             background: "transparent",
             border: "none",
             display: "flex",
             flexDirection: "row",
-            height: "100%",
+            height: "calc(100vh - 88px)", // Toolbar height
             overflowY: "visible",
             position: "relative",
-            top: typography.pxToRem(88), // Toolbar height
+            top: 0, // Toolbar height
```

Comment:
Why would this make a difference?

Reply:
Changing to persistent drawer necessitated this change to avoid a non sticky header when drawer is open

Reply:
Why? For the documentation, can you add a comment why when using persistent drawer we must use `calc(100vh - 88px)` instead of `top: 88px`?